src/DocumentationViewer.js

```diff
@@ -43,11 +38,20 @@ export default function DocumentationViewer({
     }
   });
 
-  back.connect("clicked", () => {
+  webview.get_back_forward_list().connect("changed", () => {
+    updateButtons();
+  });
```

Comment:
this is important because not only load evenets can change `can_go_back` and `can_go_forward`

If you click on a anchor like a title or something that navigates somewhere else in the current page, there is no load event but it is still possible to go back

Reply:
Ah I see, got it.